app/assets/javascripts/discourse/app/components/conditional-loading-spinner.js

```diff
@@ -2,11 +2,9 @@ import Component from "@ember/component";
 import discourseComputed from "discourse-common/utils/decorators";
 
 export default Component.extend({
-  classNameBindings: [
-    ":loading-container",
```

Comment:
That's a breaking change. I believe there are many places that rely on the fact that the whole element is wrapped in this class. For example:

```scss
// Parent element containing the users lists.
.loading-container {
  flex-basis: 100%;
}
```